packages/drawer/src/views/Drawer.tsx

```diff
@@ -214,11 +219,12 @@ export default class DrawerView extends React.PureComponent<Props> {
   private velocityX = new Value<number>(0);
   private gestureX = new Value<number>(0);
   private offsetX = new Value<number>(0);
-  private position = new Value<number>(0);
+  private position = new Value<number>(this.bigScreenSidebar ? 1 : 0);
 
   private containerWidth = new Value<number>(0);
   private drawerWidth = new Value<number>(0);
-  private drawerOpacity = new Value<number>(0);
+  // make drawer initially visible on the big screen to avoid annoying animation
+  private drawerOpacity = new Value<number>(this.bigScreenSidebar ? 1 : 0);
```

Comment:
I think we can just to `opacity: drawerType === 'sidebar' : 1 : this.drawerOpacity` below